CMakeLists.txt

```diff
@@ -408,8 +408,14 @@ if(BUILD_DEV)
     set(MIOPEN_CACHE_DIR "" CACHE STRING "")
 else()
     set(MIOPEN_BUILD_DEV 0)
-    set(MIOPEN_USER_DB_PATH "~/.config/miopen/" CACHE STRING "Default path of user db files")
-    set(MIOPEN_CACHE_DIR "~/.cache/miopen/" CACHE STRING "")
+    if(WIN32)
+        set(MIOPEN_USER_DB_PATH "$HOME\\\\.miopen\\\\db\\\\" CACHE STRING "Default path to user db files")
+        set(MIOPEN_CACHE_DIR "$HOME\\\\.miopen\\\\cache\\\\" CACHE STRING "")
+	set(MIOPEN_SYSTEM_DB_PATH "C:\\\\ProgramData\\\\MIOpen\\\\" CACHE PATH "Default path to system db files")
```

Comment:
Is it possible to use the environment variable for ProgramData here instead?

Reply:
Yes, we were thinking about the same. I will fix that.

Reply:
done